ydb/library/yql/providers/generic/connector/tests/utils/settings.py

```diff
@@ -41,26 +44,29 @@ class PostgreSQL:
 
     @classmethod
     def from_env(cls) -> 'Settings':
+        docker_compose_file = yatest.common.source_path('ydb/library/yql/providers/generic/connector/tests/docker-compose.yml')
+        endpoint_determiner = EndpointDeterminer(docker_compose_file)
+
         return cls(
             connector=cls.Connector(
                 grpc_host='localhost',
-                grpc_port=50051,
+                grpc_port=endpoint_determiner.get_port('fq-connector-go', 50051),
                 paging_bytes_per_page=4 * 1024 * 1024,
                 paging_prefetch_queue_capacity=2,
             ),
             clickhouse=cls.ClickHouse(
                 cluster_name='clickhouse_integration_test',
                 host='localhost',
```

Comment:
does connector have access to `localhost` without 
```
network_mode: host
```
in `docker-compose.yml`?

maybe we should use service name (`clickhouse`) as host and internal port (`8123` or `9000`) as port here?